index.bs

```diff
@@ -3606,7 +3605,7 @@ an [=attribution trigger=] |trigger| and an optional [=attribution source=]
 
 To <dfn>check if an [=attribution source=] and [=attribution trigger=] have matching attribution scopes</dfn> given an [=attribution source=] |source| and [=attribution trigger=] |trigger|:
 
-1. If |trigger|'s [=attribution trigger/attribution scopes=] is null, return true.
+1. If |trigger|'s [=attribution trigger/attribution scopes=] is null or [=set/is empty|empty=], return true.
```

Comment:
Can you clarify why empty trigger scopes should match everything instead of either matching nothing or only matching the source if the source's scopes are also empty? The latter seems more useful.

Reply:
The latter is equivalent to introducing a new attribution scope per discussion with @vikassahu29. the information gain computation will all be affected since `k` is increased. 

in other words, null = empty = omitted for this field.

Reply:
If `null = empty` then why does the field on the trigger structure need to be nullable at all? Let's just default the field to the empty list if it's omitted.

Reply:
Done.

Reply:
Please also update the header validator to the same effect.